### ISSUE TITLE
Publish 0.1 to crates.io

### ISSUE THREAD

**Issue by ordian** [type-metadata/type-metadata#41]
Let's discuss what's missing for the initial release on crates.io. (cc https://github.com/paritytech/parity-common/pull/312)

**Comment by ordian** [comment]
Is it correct that the 0.1 release is blocked on the currently open PRs (#36, #38, #40, #42)? 
Would be nice to have a roadmap for the initial release.
cc @ascjones

**Comment by ascjones** [comment]
Yeah would be good to have all those in before a first release. I'm working on #36 and #42 as we speak. WIll chat to @Robbepop about what else needs doing before initial release.
